doc/query.md

```diff
@@ -76,6 +81,54 @@ if (name == null) {
 
 If an index needs to be changed, first delete the existing index by calling `deleteIndexNamed(String indexName)` where the argument is the index name, then call  `ensureIndexed(List<Object> fieldNames, String indexName)` with the new definition.
 
+#### Managing a Text Index
+
+In order to perform text searches on documents in a datastore a text index needs to be created.  Our text indexing process leverages SQLite's full text search (FTS) functionality, therefore SQLite FTS needs to be available for a text index to be created.
```

Comment:
Probably be more specific about the FTS requirement:

> Our text indexing uses SQLite's full text seach, which means SQLite must have that [feature enabled at compile time][link to SQLite compile docs docs].

Do we need to note which versions of Android we've tested which have it enabled?